src/main/resources/migrations/00017_add_column_service_name_to_products.sql

```diff
@@ -0,0 +1,6 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:add_column_service_name_on_products
+ALTER TABLE products ADD COLUMN service_name VARCHAR(255);
```

Comment:
This creates a `Nullable` column. I think this should be 'NOT NULL' 
And for the records that are already in perhaps we should just make it `default 'System Generated'`

Reply:
I disagree, I think it should be `Nullable` for now. If it comes back to pay-products as blank, it shows nothing. If we default it to `System Generated` it will show `System Generated` in the top bar. Once we move this into AWS, all newly created products will have a value, there will be no `Null` values so we can add the constraint then.